dts/arm/intel_socfpga_std/socfpga.dtsi

```diff
@@ -0,0 +1,300 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2022, Intel Corporation
+ * Description:
+ * Section of the DTS file containing definitions for the HPS
+ * based on Linux, heavily modified for Zephyr
```

Comment:
Just checking, if this is based on Linux, did you relicense it from GPLv2 to Apache v2?

Reply:
We created the DTS based on our Golden System Reference Design DTS... The file is similar, as the addresses are the same, but of course, we built this one for Zephyr.